contrib/base.sh

```diff
@@ -230,20 +230,13 @@ export PY_VER_MAJOR="3.9"  # as it appears in fs paths
 # If you change PYTHON_VERSION above, update this by downloading the files manually and doing a sha256sum on it.
 export PYTHON_SRC_TARBALL_HASH="125b0c598f1e15d2aa65406e83f792df7d171cdf38c16803b149994316a3080f"
 export PYTHON_MACOS_BINARY_HASH="351fe18f4fb03be7afac5e4012fc0a51345f43202af43ef620cf1eee5ee36578"
-export DEFAULT_GIT_REPO=https://github.com/Bitcoin-ABC/ElectrumABC
-if [ -z "$GIT_REPO" ] ; then
-    # If no override from env is present, use default. Support for overrides
-    # for the GIT_REPO has been added to allows contributors to test containers
-    # that are on local filesystem (while devving) or are their own github forks
-    export GIT_REPO="$DEFAULT_GIT_REPO"
-fi
-if [ "$GIT_REPO" != "$DEFAULT_GIT_REPO" ]; then
-    # We check if it's default because we unconditionally propagate $GIT_REPO
-    # in env to _build.sh inside the docker container, and we don't want to
-    # print this message if it turns out to just be the default.
-    info "Picked up override from env: GIT_REPO=${GIT_REPO}"
+
+if [ -z "${ELECTRUM_ROOT}" ] ; then
+    PROJECT_ROOT=$(git rev-parse --show-toplevel)
+    export ELECTRUM_ROOT=${PROJECT_ROOT}
 fi
```

Comment:
All this can be written as:
```
: "${ELECTRUM_ROOT:=$(git rev-parse --show-toplevel)}"
export ELECTRUM_ROOT
```

This is shorted but also makes use of double quotes, which will prevent your script from failing should there be a space in the path.